mlir/test/Dialect/SparseTensor/sparse_matmul_one.mlir

```diff
@@ -0,0 +1,75 @@
+// RUN: mlir-opt %s --linalg-generalize-named-ops \
+// RUN:             --sparsification-and-bufferization | FileCheck %s
+
+#CSR_ones_complex = #sparse_tensor.encoding<{
+  map = (d0, d1) -> (d0 : dense, d1 : compressed)
+// explicitVal = (1.0, 0.0) : complex<f32>,
+// implicitVal = (1.0, 0.0) : complex<f32>
```

Comment:
(0.0, 0.0) for implicit value? I'm planing to add verification to fail anything else for complex implicit value.

Reply:
Of course, I was playing around with values and ended up with the wrong one ;-)